composer.json

```diff
@@ -11,11 +11,13 @@
     ],
     "require": {
         "php": ">=7.0",
+        "nesbot/carbon": "~1.20",
         "symfony/var-dumper": "~3.3"
     },
     "require-dev": {
         "mockery/mockery": "~1.0",
-        "phpunit/phpunit": "~6.0"
+        "phpunit/phpunit": "~6.0",
+        "symfony/var-dumper": "~3.3"
```

Comment:
We don't need this, since it's in require now.

Reply:
Is it? This is it on my end:

![image](https://user-images.githubusercontent.com/3182864/35657365-3af6c38c-06e3-11e8-9dc7-9d1de7783d05.png)

Reply:
Right... `symfony/var-dumper` is in both so we don't need it in `require-dev`, right?

Reply:
We don't, removed.